### PR TITLE
Code Review com_fields frontend

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes
- Change `defined('_JEXEC') or die();` to `defined('_JEXEC') or die;`
- Change `if (! $item)` to `if (!$item)` and similiar
- some general Codestyle / Code review
- remove one redundant else ;)

### CODE REVIEW

File: components/com_fields/controllers/field.php [hunk 1]
```diff
@@ -6,23 +6,35 @@
...
+ * Fields controller class for Fields Component.
+ *
+ * @since  3.6
+ */
 class FieldsControllerField extends JControllerLegacy
 {
-
-	public function catchange ()
+	/**
+	 * Some Doku here
```
Comment (wilsonge) [inline components/com_fields/controllers/field.php]: :P

**Comment by wilsonge** [comment]
Remove/fix the funny comment and we can squash the commits and merge this :)

**Comment by zero-24** [comment]
done with the last commit. https://github.com/joomla-projects/custom-fields/pull/10/commits/a2e4c37dfbde23b77c168b104456593f8ea67ed1

**Comment by wilsonge** [comment]
Will squash commits + merge tomorrow morning :)

**Comment by zero-24** [comment]
:+1:

**Comment by laoneo** [comment]
Thanks @wilsonge for all the help here!

**Comment by wilsonge** [comment]
Merged with https://github.com/joomla-projects/custom-fields/commit/d3550c98d645d22fb10a7c53abf2a28aa6adbf59

**Comment by zero-24** [comment]
:+1:

**Pull request closed without merging**